script/vsts/platforms/windows.yml

```diff
@@ -27,6 +27,9 @@ jobs:
   - script: |
       ECHO Installing npm-windows-upgrade
       npm install --global --production npm-windows-upgrade
+    displayName: Install npm-windows-upgrade
+
+  - script: |
```

Comment:
I had to split this step into two separate steps. Prior to this change, the first command was getting executed, but the second command was _not_ getting executed (for some reason).

## Before (One step)

https://github.visualstudio.com/Atom/_build/results?buildId=38113&view=logs&jobId=97a617bf-bcbd-5dfa-bba2-cfba2747b693&taskId=ff3b67c9-4d3a-551a-4d02-46b3dfea0bbc&lineStart=12&lineEnd=16&colStart=1&colEnd=40

<img width="953" alt="Screen Shot 2019-04-18 at 1 03 36 PM" src="https://user-images.githubusercontent.com/2988/56378611-9100ed80-61db-11e9-86ff-4222f0300469.png">

## After (Two steps)

<img width="982" alt="Screen Shot 2019-04-18 at 1 11 19 PM" src="https://user-images.githubusercontent.com/2988/56378613-9100ed80-61db-11e9-8320-3cd2b0b2821d.png">
<img width="972" alt="Screen Shot 2019-04-18 at 1 05 34 PM" src="https://user-images.githubusercontent.com/2988/56378612-9100ed80-61db-11e9-8a68-782ba4ae19b3.png">

---

/fyi @smashwilson